packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/migrations/index.ts

```diff
@@ -6,9 +6,17 @@
  * Side Public License, v 1.
  */
 
+import { flow } from 'lodash';
+import { default as get } from 'lodash/fp/get';
 import { TransformFn } from '../types';
 import { transformMigrationVersion } from './transform_migration_version';
+import { transformSetManagedDefault } from './transform_set_managed_default';
 
 export const migrations = {
-  '8.8.0': transformMigrationVersion,
+  '8.8.0': flow(
+    transformMigrationVersion,
+    // extract transformedDoc from TransformResult as input to next transform
+    get('transformedDoc'),
```

Comment:
Yea, you're right, there is a risk that we return `additionalDocs` and I could work that in. However, I synced with @dokmic on that and the risk is low.

Reply:
Yeah, we could implement our own chaining HOC here to properly handle `additionalDocs` instead of just using `lodash.flow`, but it seems fine not to.